src/GlobalExports.ts

```diff
@@ -27,3 +27,12 @@ export function lazyExport(component: IComponentDefinition, promiseResolve: Func
   Initialization.registerAutoCreateComponent(component);
   promiseResolve(component);
 }
+
+export function lazyExportModule(mod: any, promiseResolve: Function) {
```

Comment:
Actually, I should remove the error.

It should not throw, nor force a specific type.